WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -1,29 +1,65 @@
 import Foundation
 import UIKit
 import SwiftUI
+import Combine
 
 /// View model for `HubMenu`.
 ///
 final class HubMenuViewModel: ObservableObject {
 
-    let siteID: Int64
+    @Published private(set) var siteID: Int64
```

Comment:
I don't think making `siteID` an `@Published` property is necessary, since we're already creating a new `HubMenu` and its view model whenever switching the default store. WDYT?

Reply:
Updated to `let` 👍